examples/classification.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
```

Comment:
I seem to have forgot to delete an old example. 
This is from before we had a common pipeline and stage interface, and should probably be removed to avoid confusion.